homeassistant/components/google/manifest.json

```diff
@@ -4,7 +4,7 @@
   "config_flow": true,
   "dependencies": ["application_credentials"],
   "documentation": "https://www.home-assistant.io/integrations/calendar.google/",
-  "requirements": ["gcal-sync==4.1.2", "oauth2client==4.1.3"],
+  "requirements": ["gcal-sync==4.1.2"],
```

Comment:
Google is using oauth2client.

Reply:
Not anymore. I inlined the 2 constants. Nobody is using oauth2client now.

Reply:
There's still this:

https://github.com/home-assistant/core/blob/dev/homeassistant/components/google/api.py#L11-L17

Reply:
Sorry.  My VS Code failed to find that usage.
I'm closing this PR since I sent https://github.com/home-assistant/core/pull/85785

Reply:
Sorry.  My VS Code failed to find that usage.
I'm closing this PR since I sent https://github.com/home-assistant/core/pull/85785